origami.json

```diff
@@ -1,25 +1,25 @@
 {
-    "description": "Financial Times icons",
-    "origamiType": "module",
-    "origamiVersion": 1,
-    "support": "https://github.com/Financial-Times/o-ft-icons/issues",
-    "supportStatus": "active",
-    "browserFeatures": {
-        "required": [
-            "fontface"
-        ]
-    },
-    "ci": {
-        "circle": "https://api.travis-ci.org/repos/Financial-Times/o-ft-icons/builds.json"
-    },
-    "demos": [
-        {
-            "name": "main",
-            "template": "demos/src/demo.mustache",
-            "sass": "demos/src/demo.scss",
-            "data": "demos/src/data.json",
-            "expanded": true,
-            "description": ""
-        }
-    ]
+  "description": "Financial Times icons",
```

Comment:
Just a couple of indent inconsistencies, L2 here, and L9 are still using spaces. Sorry to be so fussy